pokepro/src/components/Wrapper/Wrapper.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+type Props = {
+    children: string | JSX.Element | JSX.Element[] 
+  }
+
+function Wrapper(children:Props) {
```

Comment:
This component seems to just return children, If its a WIP component then add a //TODO comment and explain what is it for, if its not required then we can delete this component.